crates/ironrdp-rdpdr/src/backend/mod.rs

```diff
@@ -6,9 +6,10 @@ use crate::pdu::{
 };
 use core::fmt;
 use ironrdp_pdu::PduResult;
+use ironrdp_svc::AsAny;
 
 /// OS-specific device redirection backend inteface.
-pub trait RdpdrBackend: fmt::Debug + Send {
+pub trait RdpdrBackend: AsAny + fmt::Debug + Send + 'static {
```

Comment:
Nit: it’s technically okay since we currently only use this trait for boxed trait-objects, but I would tend to add the `'static` bound at the place where it’s necessary in order to not over-constrain the definition. For instance:
```rust
    pub fn into_box<T>(backend: T) -> Box<dyn RdpdrBackend>
    where
        T:  RdpdrBackend + 'static, // <- here
    {
        Box::new(backend)
    }

```
Or is there is another reason for it that I’m missing?

Reply:
It's a requirement of for `T` of `downcast_ref::<T>`, I've moved it there in f7c14ea1ea77713c7b502a73559294c371bd2619

Reply:
Oh right. My bad. [`Any`](https://doc.rust-lang.org/std/any/trait.Any.html) requires `'static` in the first place, so we might as well add this bound on `AsAny`. I pushed a commit for that: https://github.com/Devolutions/IronRDP/pull/234/commits/6286dd70038c65ffbc47ca5199930c5ef31318f5